src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,49 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { INSET_BORDER } from '@utilities/borderStyle';
+import { merge } from '@utilities/merge';
+import { AnimatePresence, motion } from 'framer-motion';
+import React, { ReactElement } from 'react';
+import { ToastAnimationDirection, ToastProps, toastStylesBackgroundColorsMap } from './types';
+
+const CONTAINER_BASE_CLASSES = 'tw-min-h-[4.25rem] tw-bg-base tw-rounded-lg tw-shadow-mid tw-z-[120000]';
+const CONTAINER_CLASSES = merge([CONTAINER_BASE_CLASSES, INSET_BORDER]);
```

Comment:
I know it was repeated in the Menu and Flyout component, but to avoid duplicated code, I would move those classes to utilities.

Reply:
`INSET_BORDER` from `@utilities/borderStyle` is used in:
- `Overlay.tsx`
- `Menu.tsx`
- `Toast.tsx`

In `Overlay.tsx` it is merged with:
- `tw-flex tw-flex-col tw-flex-auto tw-min-h-0 tw-rounded
`


In `Menu.tsx` it is merged with:
- `tw-relative tw-bg-base tw-rounded tw-py-2 tw-shadow-mid tw-z-[120000]
`


And i merged it in the `Toast.tsx` with:
- `tw-min-h-[4.25rem] tw-bg-base tw-rounded-lg tw-shadow-mid tw-z-[120000]
`

So it looks like there is no class that is used in all 3 components (except  INSET_BORDER which is already in utilities).
So i guess it does not make any sense to move something to utilities right?

Reply:
Mmm it's not really consistent. Ok, don't worry about that, we'll take care.
This will be standardized when we work on the new `Container` component. 😉